src/plugins/sensor/AirSimSensor/AirSimSensor.cpp

```diff
@@ -298,6 +307,9 @@ void AirSimSensor::request_images() {
         running_mutex_.lock();
         running = running_;
         running_mutex_.unlock();
+
```

Comment:
Right now scrimmage runs well below wall time when running with airsim. This was a quick attempt to reduce the number of calls to airsim, which I believe was the bottleneck. This helped, but I think more important was the elimination of multiple sends of the same image. There is probably a better way to do this. One thing we might look into is to use the "computer vision" mode in airsim

Reply:
Computer Vision mode turns off the AirSim physics engine. Scrimmage is setting the pose, but is not doing the full job of a physics engine. Also in the future we would like to get collisions out of AirSim. I believe that is why we are not using computer vision mode right now.

Reply:
Great catch on the images being published multiple times! This will help keep the simulation smooth when we start simulating with multiple vehicles. Thank you!